File: Main/ReferenceDLLs/Pack1/Facebook Visual Kit VB/FVK Source Code/FVK/FVK/ComponentParts/IJSONSerialize.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

Namespace FVK
    ''' <summary>
    ''' This interface uses for convesion from object data to JSON format
    ''' </summary>
    Public Interface IJSONSerialize
        ''' <summary>
        ''' Serialize object to JSON format
        ''' </summary>
        ''' <returns>JSON format string</returns>
        Function JSONSerialize() As String
    End Interface
End Namespace

File: Main/ReferenceDLLs/Pack1/Facebook Connect Visual Kit CS/FVK Source Code/FVK/FVK/ComponentParts/IJSONSerialize.cs
// WARNING: 
// The source code of this class can be used and modified to your particular needs. 
// Sharing the class with unregistered users or using it with unregistered websites 
// or Facebook applications is not allowed and violates the rights of intellectual 
// property. 

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace FVK
{
    /// <summary>
    /// This interface uses for convesion from object data to JSON format
    /// </summary>
    public interface IJSONSerialize
    {
        /// <summary>
        /// Serialize object to JSON format
        /// </summary>
        /// <returns>JSON format string</returns>
        string JSONSerialize();
    }
}
